Sources/OpenCombine/AnySubscriber.swift

```diff
@@ -231,24 +231,28 @@ internal final class SubjectSubscriber<Downstream: Subject>
 {
     internal var downstreamSubject: Downstream?
     internal var upstreamSubscription: Subscription?
+    private var isCompleted = false
```

Comment:
I wonder if instead of adding a new flag we can check if `downstreamSubject` is nil, and set it to nil upon completion. I may be wrong here though.